docs/reference/client-worker-configuration.md

```diff
@@ -31,19 +31,23 @@ The file may be called by any name but should have the following structure:
     "windows": {
       "localBuildDirectory": "C:\project\bin\windows"
     },
-    "macos": {...},
-    "linux": {...}
+    "macos": {
+      ...
+    },
+    "linux": {
+      ...
+    }
   }
 }
 ```
 
 | Field | Required/Optional | Description | 
 | :------------- | :------------- | :------- |
-| `workerType` | Required | Used to identify this worker type elsewhere in your project. |
-|`layer`| Optional | The name of the simulation layer of this worker, used to determine the load balancing strategy. For more information, see [introduction to layers](https://docs.improbable.io/reference/latest/shared/worker-configuration/layers#introduction-to-layers).|
+| `workerType` | Required | Used to identify this worker type elsewhere in your project. <br><br> **Note:** A `workerType` must be unique project-wide between server and client workers. The `workerType` specified in this field must also match the `workerType` that your worker reports in its `ConnectionParameters` when connecting to a deployment. For more information, see the [API reference](https://docs.improbable.io/reference/latest/csharpsdk/api-reference#improbable-worker-connectionparameters-class). |
+|`layer`| Optional | The name of the simulation layer of this worker, used to determine the load balancing strategy. For more information, see [introduction to layers](https://docs.improbable.io/reference/latest/shared/worker-configuration/layers#introduction-to-layers). |
```

Comment:
```suggestion
|`layer`| Optional | The name of the simulation layer of this worker type, used to determine the load balancing strategy. For more information, see [Layers](https://docs.improbable.io/reference/latest/shared/worker-configuration/Layers). |
```

Updated the link because I changed this yesterday with the release of Kraken :)

Reply:
Nice. Done!